### PR TITLE
included description of setFeature function

### PR DESCRIPTION
**Pull request by lajunek**
A code showing the usage of the setFeature function used in atlas feature

### Description

Goal:

Ticket: #

- [ ] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by lajunek** [comment]
@DelazJ @SrNetoChan can your review my changes?

File: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst [hunk 1]
```diff
@@ -51,7 +51,26 @@ panel provides the following functionalities (see figure_layout_html_ppt_):
   override` button, to provide a URL from the contents of an attribute field of
   a table or using a regular expression.
 * In :guilabel:`Source` you can enter text in the textbox with some HTML tags or
-  provide a full HTML page.
+  provide a full HTML page. Provided that you created a relation described in
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]: I'd skip all the verbal reference to the section with adding a hyperlink
```suggestion
  provide a full HTML page. Provided that you created a :ref:`relation between
  layers <vector_relations>` a simple code including
```

File: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst [hunk 2]
```diff
@@ -51,7 +51,26 @@ panel provides the following functionalities (see figure_layout_html_ppt_):
...
+  in :ref:`Creating one or many to many relations<_vector_relations>` a simple code including
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]:
```suggestion
```

File: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst [hunk 3]
```diff
@@ -51,7 +51,26 @@ panel provides the following functionalities (see figure_layout_html_ppt_):
...
+  HTML, JavaScript and GeoJSON gives you full access to the parent and child features. 
+  This way, the atlas feature browses through the parent layers and and creates a page
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]:
```suggestion
  This way, the atlas feature browses through the parent layers and creates a page
```

File: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst [hunk 4]
```diff
@@ -51,7 +51,26 @@ panel provides the following functionalities (see figure_layout_html_ppt_):
...
+  for every existing relation, depicting a parent feature and the relating child features
+  on one page each.
```
Comment (DelazJ) [inline source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]: not sure I understand what happens

Comment (lajunek) [reply source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]: @DelazJ I added more code and tried to describe what happens in there !

**Comment by lajunek** [comment]
@DelazJ ok, All checks have failed. Can we fix this problem?

File: source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst [hunk 5]
```diff
@@ -51,7 +51,32 @@ panel provides the following functionalities (see figure_layout_html_ppt_):
...
+      function setFeature(feature)
+               {
+                document.getElementById('name').innerHTML = feature.properties.NAME;
+                document.getElementById('my_children').innerHTML = '';
+                feature.properties.my_relation.forEach(function(child_feature)
+                                       {
+                                       var node = document.createElement("li"):
+                                       node.appendChild(document.createTextNode(child_feature.NAME));
+                                       document.getElementById('my_children').appendChild(node);
+                                       }
```
Comment (havatv) [inline source/docs/user_manual/print_composer/composer_items/composer_html_frame.rst]: Missing end parenthesis for "forEach"?

**Comment by DelazJ** [comment]
🙄

**Comment by SrNetoChan** [comment]
Hi @lajunek not sure what happened in here. Did you give up on including this information? Was it because of my comment?

Anyway, merging this had no pratical impact because it did not change anything. You could either close the PR without merging, or move the text to another file in the same branch, that way updating this PR.

**Pull request merged**